ubtue_Peeters.js

```diff
@@ -201,8 +201,6 @@ function scrape(doc, url) {
 		if (match)
 			item.date = match[1];
 	}
-
```

Comment:
There seems to be only one `item.complete()` in line 195 any more, but this is encolsed in an  `if` condition so the mandatory `item.complete()` will be missing in cases where the `if` does not match.

Reply:
i changed it. thanks!